examples/with-ant-design/pages/_app.js

```diff
@@ -0,0 +1,5 @@
+import 'antd/dist/antd.css';
```

Comment:
I prefer to use a minified file.
```suggestion
import 'antd/dist/antd.min.css';
```

Reply:
We automatically minify the CSS. It should not be imported already minified.